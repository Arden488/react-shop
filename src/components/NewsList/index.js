import React from 'react';

import NewsListItem from '../NewsListItem';

const NewsList = () => (
  <div>
    <NewsListItem code="1" title="1" />
    <NewsListItem code="2" title="2" />
    <NewsListItem code="3" title="3" />
    <NewsListItem code="4" title="4" />
    <NewsListItem code="5" title="5" />
  </div>
);

export default NewsList;
