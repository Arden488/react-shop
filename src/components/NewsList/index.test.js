/* global it, expect, shallow, beforeEach */

import React from 'react';

import NewsList from './index';

let component = null;

beforeEach(() => {
  component = shallow(<NewsList />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
