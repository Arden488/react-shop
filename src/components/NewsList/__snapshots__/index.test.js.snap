// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <NewsListItem
    code="1"
    title="1"
  />
  <NewsListItem
    code="2"
    title="2"
  />
  <NewsListItem
    code="3"
    title="3"
  />
  <NewsListItem
    code="4"
    title="4"
  />
  <NewsListItem
    code="5"
    title="5"
  />
</div>
`;
